controllers/hcpvaultsecretsapp_controller.go

```diff
@@ -288,6 +299,57 @@ func (r *HCPVaultSecretsAppReconciler) Reconcile(ctx context.Context, req ctrl.R
 	}, nil
 }
 
+func (r *HCPVaultSecretsAppReconciler) startShadowSecretCleanupRoutine(ctx context.Context) {
+	logger := log.FromContext(ctx)
+	// cleanup of orphaned shadow secrets runs once every hour
+	ticker := time.NewTicker(1 * time.Hour)
+
+	for {
+		select {
+		case <-ticker.C:
```

Comment:
Should maybe add a ctx done case to know when to close this loop out.

Reply:
@lursu I believe that the loop is supposed to run indefinitely to check if any orphaned secrets exist at any point, right?

Reply:
Discussed with @lursu during our sync today and we believe that it would be good to add a comment here to make sure others know that this is a long-running process.